code/Dev/State/DisableVacuumState.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace SilverStripe\PostgreSQL\Dev\State;
+
+use SilverStripe\Core\Config\Config;
+use SilverStripe\Dev\SapphireTest;
+use SilverStripe\Dev\State\TestState;
+use SilverStripe\PostgreSQL\PostgreSQLSchemaManager;
+
+class DisableVacuumState implements TestState
+{
+    /**
+     * Called on setup
+     *
+     * @param SapphireTest $test
+     */
+    public function setUp(SapphireTest $test)
+    {
+        // TODO: Implement setUp() method.
+    }
+
+    /**
+     * Called on tear down
+     *
+     * @param SapphireTest $test
+     */
+    public function tearDown(SapphireTest $test)
+    {
+        // TODO: Implement tearDown() method.
```

Comment:
I assume you won't be implementing it, perhaps say no-op instead?

Reply:
I force pushed but never actually added the changes 😅